src/plugins/management_overview/public/plugin.ts

```diff
@@ -65,6 +66,9 @@ export class ManagementOverViewPlugin
       icon: '/ui/logos/opensearch_mark.svg',
       order: 9000,
       category: DEFAULT_APP_CATEGORIES.management,
+      navLinkStatus: coreSetup.chrome.navGroup.getNavGroupEnabled()
+        ? AppNavLinkStatus.hidden
+        : AppNavLinkStatus.visible,
```

Comment:
nit: does AppNavLinkStatus only have 2 states? If yes, looks like we can use boolean? If not, what are the other states?

Reply:
There are default and disabled. It is a existing enum and mechanism to hide applications from left navigation.